frontend/views/containers/chatroom/ChatMain.vue

```diff
@@ -1184,7 +1126,38 @@ export default ({
         e?.dataTransfer.files?.length &&
           this.$refs.sendArea.fileAttachmentHandler(e?.dataTransfer.files, true)
       }
-    }
+    },
+    processSwitchQueue: debounce(async function () {
+      if (this.ephemeral.chatroomSwitchQueue.length === 0) return
+
+      // Take the most recent chatroom entry on the queue and discard everything else. This way we can speed up the process of switching chatrooms.
+      const targetChatroomId = this.ephemeral.chatroomSwitchQueue.pop()
+      this.ephemeral.chatroomSwitchQueue = []
+      this.ephemeral.renderingChatRoomId = targetChatroomId
+
+      try {
+        await this.initializeState(true)
```

Comment:
After releasing v1.2.16 with these changes, I'm noticing that chatrooms are loading a bit more slowly than before upon switching between them.

I was trying to figure out what change could be causing this, and noticed that previously this call was called using `this.initializeState()` in `setInitMessages`.

I'm wondering if this could be the cause of the slowness. @SebinSong, do you know why `true` is now being passed in? Does it need to be passed in? Or would not passing in fix the slowness I'm seeing?